raiden-derive/src/ops/batch_delete.rs

```diff
@@ -92,6 +92,8 @@ pub(crate) fn expand_batch_delete(
         }
     };
 
+    let tracing_span = super::tracing_inner_run_span!("batch_write_item");
```

Comment:
Create token stream to call DynamoDB API with span.  
if feature `tracing` is not active, token stream will be created without making span.